File: Python/1043_Triangle.py
x, y, z = map(float, input().split())

if(x + y > z and x + z > y and z + y > x):
    print(f"Perimetro = {x+y+z:.1f}")
else:
    print(f"Area = {((x + y) * z) / 2:.1f}")

File: C/1043_Triangle.c
#include <stdio.h>
#include <math.h>

int main() {
  float a, b, c;

  scanf("%f %f %f", &a, &b, &c);

  if (a + b > fabs(c) && a + c > fabs(b) && b + c > fabs(a)) {
    printf("Perimetro = %.1f\n", a + b + c);
  } else {
    printf("Area = %.1f\n", 0.5 * (a + b) * fmin(c, fabs(a - b)));
  }

  return 0;
}